### PR TITLE
Update API doc with media supported configuration keys

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: Documentation/api-reference.md [hunk 1]
```diff
@@ -336,6 +346,17 @@ About the XDM data structure, please refer to the [starting the session
...
 
+- `@param xdmData as object : the XDM data of type "media.sessionStart"`
+- `@param configuration as object : the session-level configurationr`
+
+##### Configuration Keys
+
+| Constants | Raw value | Required |
+| :-- | :--: | :--: |
+| `ADB_CONSTANTS.MEDIA_SESSION_CONFIGURATION.CHANNEL` | "config.channel" | **No**
+| `ADB_CONSTANTS.MEDIA_SESSION_CONFIGURATION.AD_PING_INTERVAL` | "config.adpinginterval" | **No**
```
Comment (praveek) [inline Documentation/api-reference.md:357]: Can you also add a little description on the value type and allowed range?

File: Documentation/api-reference.md [hunk 2]
```diff
@@ -336,6 +346,17 @@ About the XDM data structure, please refer to the [starting the session
 createMediaSession: function(xdmData as object, configuration = {} as object) as void
 ```
 
+- `@param xdmData as object : the XDM data of type "media.sessionStart"`
+- `@param configuration as object : the session-level configurationr`
+
```
Comment (addb) [inline Documentation/api-reference.md:351]: Add a note something on these lines:

> [!NOTE}
> If ping interval is not set default interval `10 sec` will be used.

File: Documentation/api-reference.md [hunk 3]
```diff
@@ -336,6 +346,20 @@ About the XDM data structure, please refer to the [starting the session
 createMediaSession: function(xdmData as object, configuration = {} as object) as void
 ```
 
+- `@param xdmData as object : the XDM data of type "media.sessionStart"`
+- `@param configuration as object : the session-level configurationr`
```
Comment (praveek) [inline Documentation/api-reference.md:350]: Typo in configuration

File: Documentation/api-reference.md [hunk 4]
```diff
@@ -336,6 +346,20 @@ About the XDM data structure, please refer to the [starting the session
...
+> [!NOTE]
+> If the ping interval is not set, the default interval of `10 sec` will be used.
+
+##### Configuration Keys
+
+| Constants | Raw value | Type | Range | Required |
+| :-- | :--: | :--: | :--: | :--: |
+| `ADB_CONSTANTS.MEDIA_SESSION_CONFIGURATION.CHANNEL` | "config.channel" | String | | **No**
+| `ADB_CONSTANTS.MEDIA_SESSION_CONFIGURATION.AD_PING_INTERVAL` | "config.adpinginterval" | Integer | 1~10 | **No** 
+| `ADB_CONSTANTS.MEDIA_SESSION_CONFIGURATION.MAIN_PING_INTERVAL` | "ig.mainpinginterval" | Integer | 10~50 | **No**
```
Comment (praveek) [inline Documentation/api-reference.md:361]: Typo config.mainpinginterval

**Review by praveek — CHANGES_REQUESTED** [review]

**Review by praveek — APPROVED** [review]

**Review by praveek — APPROVED** [review]

**Pull request merged**